src/mount.ts

```diff
@@ -20,16 +26,80 @@ export type MountOptions = {
   prepareContainer?: (container: HTMLElement) => void;
 };
 
+export type EnzymePropSelector = {
+  [key: string]: any;
+};
+
+export type ComponentType<Props> =
+  | ComponentClass<Props>
+  | FunctionComponent<Props>;
+
+export type EnzymeSelector = string | ComponentType<any> | EnzymePropSelector;
```

Comment:
Do we actually use prop selectors (`EnzymePropSelector`) anywhere?

Reply:
Now that you mention it, no, I haven't seen any use.